log/zap.go

```diff
@@ -148,6 +178,28 @@ func (l Log) WithOptions(opts ...zap.Option) Log {
 	}
 }
 
+const event_key = "event"
+
+func (fl fieldLogger) EventInfo(msg string, fields ...Field) {
```

Comment:
you can create one function called "Event" and concate it with .Info() and so on... no need for these functions

Reply:
fixed